SpywareFilter/sections/specific.txt

```diff
@@ -391,7 +391,7 @@ culture.ru###page-error-modal
 ||posty.kr/api/tr|
 ||rfksrv.com/rfk/$third-party
 ||rfksrv.com/rfkj/$image
-/emirates.com\/.*\/[A-Za-z0-9]{4,}$/$domain=emirates.com
+/emirates.com\/[A-Za-z0-9\/]{45,55}\/[A-Za-z0-9]{4,}$/$domain=emirates.com
```

Comment:
The request from the [issue ](https://github.com/AdguardTeam/AdguardFilters/issues/104312) won't be blocked if you change the rule.
What exactly was blocked wrongly?

Reply:
Got it, thank you.
Btw, the rule should be adjusted because it does not block such requests
```
https://www.emirates.com/4tBkPc6iO/i6vXClVtw/Oa3pLfwJ/bTZxKC1lBA/cA/8iYis7cl4
```